### PR TITLE
fix tiny typo in index.md

### PR DESCRIPTION
**Pull request by matsluni**
just a tiny typo

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -28,4 +28,4 @@ Live demo is available on [http://bootzooka.softwaremill.com](http://bootzooka.s
 
 ## License
 
-Project is licensed under [Apache 2.0 License](http://www.apache.org/licenses/LICENSE-2.0.html) which means you can freely use any part of the project.
```
Comment (kciesielski) [inline index.md]: Thanks for fixing the typo above :) What exactly changed in this line?

Comment (matsluni) [inline index.md]: Hmm... I didn't change the line. Maybe be some side effect from GH online editing. Sorry for didn't noticing.

I will update the PR.

**Comment by matsluni** [comment]
I updated the PR.

**Comment by kciesielski** [comment]
Thanks :)

**Pull request merged**